src/PVRIptvData.cpp

```diff
@@ -66,6 +74,23 @@ inline bool GetNodeValue(const xml_node<Ch> * pRootNode, const char* strTag, std
   return true;
 }
 
+template<class Ch>
+inline bool GetJoinedNodeValues(const xml_node<Ch>* pRootNode, const char* strTag, std::string& strStringValue)
+{
+  xml_node<Ch>* pChildNode = nullptr;
+  for (pChildNode = pRootNode->first_node(strTag); pChildNode; pChildNode = pChildNode->next_sibling(strTag))
```

Comment:
As `pChildNode` is not used outside the for loop I would put its declaratiom in the `for` statement.